src/main/java/com/github/stephengold/joltjni/AaBox.java

```diff
@@ -396,12 +467,101 @@ public AaBox transformed(Mat44Arg matrix) {
 
         return result;
     }
+
+    /**
+     * Get surface area of bounding box.
+     *
+     * @return the area
+     */
+    @Override
+    public float getSurfaceArea() {
+        long boxVa = va();
+        float result = getSurfaceArea(boxVa);
+        return result;
+    }
+
+    /**
+     * Check if this box overlaps with another box.
+     *
+     * @param other the other box to check
+     *
+     * @return {@code true} if they overlap, otherwise {@code false}
+     */
+    @Override
+    public boolean overlaps(ConstAaBox other) {
+        long boxVa = va();
+        long inOtherVal = other.targetVa();
+        targetVa();
+        boolean result = overlaps(boxVa, inOtherVal);
+        return result;
+    }
+
+    /**
+     * Check if this box overlaps with a plane.
+     *
+     * @param plane the {@code Plan} object to be checked
+     *
+     * @return {@code true} if they overlap, otherwise {@code false}
+     */
+    @Override
+    public boolean overlaps(ConstPlane plane) {
+        long boxVa = va();
+        float pc = plane.getConstant();
+        float pnx = plane.getNormalX();
+        float pny = plane.getNormalY();
+        float pnz = plane.getNormalZ();
+
+        boolean result = overlaps(boxVa, pc, pnx, pny, pnz);
+        return result;
+    }
+
+    /**
+     * Calculate the support vector for this convex shape.
+     *
+     * @param direction the direction vector
+     *
+     * @return the support vector
+     */
+    @Override
+    public Vec3 getSupport(Vec3Arg direction) {
```

Comment:
ConstAaBox methods should be arranged in alphabetical order within this section of the file